bridge-token-factory/src/lib.rs

```diff
@@ -75,6 +78,48 @@ pub enum Role {
     UnrestrictedDeposit,
     UnrestrictedDeployBridgeToken,
     MetadataManager,
+    FeeSetter,
+    FeeClaimer,
+}
+
+#[derive(
+    Default, BorshDeserialize, BorshSerialize, Debug, Clone, Serialize, Deserialize, PartialEq,
+)]
+pub struct DepositTokenBounds {
+    lower_bound: u128,
+    upper_bound: u128,
+}
+
+#[derive(
+    Default, BorshDeserialize, BorshSerialize, Debug, Clone, Serialize, Deserialize, PartialEq,
+)]
+pub struct WithdrawTokenBounds {
+    lower_bound: u128,
+    upper_bound: u128,
+}
+
+#[derive(
+    Default, BorshDeserialize, BorshSerialize, Debug, Clone, Serialize, Deserialize, PartialEq,
+)]
+pub struct DepositFeePercentage {
+    eth_to_near: u128,
+    eth_to_aurora: u128,
+}
+
+#[derive(
+    Default, BorshDeserialize, BorshSerialize, Debug, Clone, Serialize, Deserialize, PartialEq,
+)]
+pub struct WithdrawFeePercentage {
+    near_to_eth: u128,
+    aurora_to_eth: u128,
+}
```

Comment:
Why do you need regular `Serialize` and `Deserialize`?

Reply:
The build fails if this is not included :(